zitadel/README.md

```diff
@@ -214,12 +214,12 @@ Grant role `isv` of the project `eclipse-xpanse` to the new isv user.
 
 #### Edit Metadata Of New Isv User
 
-Edit the metadata of the new isv user. Add key `namespace` with fixed value into the metadata.
-![edit_isv_user_metadata_add_namespace](static/edit_isv_user_metadata_add_namespace.png)
+Edit the metadata of the new isv user. Add key `serviceVendor` with fixed value into the metadata.
+![edit_isv_user_metadata_add_serviceVendor](static/edit_isv_user_metadata_add_serviceVendor.png)
```

Comment:
According to the requirements, the field set in IAM should be `isv` not `serviceVendor`
![image](https://github.com/user-attachments/assets/0749bbb8-ff73-4f19-bdc0-2e069f1142fe)

Reply:
fixed.